<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>784</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>781</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="packageView">
     <attribute name="title">
      <string>Package List</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>761</width>
        <height>531</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="packageTable"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPackageMainButton">
         <property name="text">
          <string>Add Package</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="signoutPackageButton">
         <property name="text">
          <string>Sign-Out Package</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="removePackageButton">
           <property name="text">
            <string>Remove Packages</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="clearTableButton">
           <property name="text">
            <string>Clear Table</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="addPackage">
     <attribute name="title">
      <string>Add Package</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>169</x>
        <y>49</y>
        <width>401</width>
        <height>411</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Recipient:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="recipientNameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Courier:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="courierComboBox">
         <item>
          <property name="text">
           <string>Canada Post</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DHL</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>FedEx</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Purolator</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UPS</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Tracking Number:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="trackingNumberEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTextEdit" name="descriptionEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="addPackageButton">
         <property name="text">
          <string>Add Package</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Package">
   <property name="text">
    <string>Add Package</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
